endpoint_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	"reflect"
 	"testing"
 
-	"github.com/elliotchance/sshtunnel"
+	"github.com/jamesgdiaz/sshtunnel"
```

Comment:
When you fork a package you should clone it to the path of the original package so that you don't have to change these.